docs/src/pages/ButtonsPage.js

```diff
@@ -35,28 +32,40 @@ const ButtonsPage = () => (
         </ReactPlayground>
       </Col>
       <h4 className='col s12'>
```

Comment:
What about an example of a Flat button, corresponding to the one in `materializecss`?

Reply:
Done.